firefox-ios/Client/Frontend/Browser/MainMenu/Redux/MainMenuState.swift

```diff
@@ -14,6 +14,16 @@ struct AccountData: Equatable {
     let iconURL: URL?
 }
 
+struct TelemetryInfo: Equatable {
+    let isHomepage: Bool
+    let isActionOn: Bool?
+
+    init(isHomepage: Bool, isActionOn: Bool? = nil) {
+        self.isHomepage = isHomepage
+        self.isActionOn = isActionOn
+    }
```

Comment:
nit, not really needed in a struct given we have default initalizer here and you're not doing anything special in the init.

Reply:
huh. It should be nil, given that it's an optional. Anyway, I wouldn't worry about it. this is fine.